Documentation/ceph-disaster-recovery.md

```diff
@@ -237,6 +237,116 @@ kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `mon.count`.
 
+## Restoring Mon Quorum using healthy OSDs
+
+If all mons fail, it might be possible to recover the monstore from the OSDs. [Ceph's documentation](https://docs.ceph.com/en/latest/rados/troubleshooting/troubleshooting-mon/#recovery-using-osds) describes how to do it.
+
+> **WARNING**: Please keep in mind that recovery is not guaranteed for the cluster or all data, even if these steps succeed. We recommend restoring your data from backup if available.
+
+### Prerequisites
+
+[toolbox](./ceph-toolbox.md) is deployed.
+
+### Preparation
+
+* Take backup of deployments of mons, osds, and toolbox.
+* Stop operator
+
+```console
+NS=<cluster's namespace such as "rook-ceph">
+kubectl -n ${NS} scale deployment rook-ceph-operator --replicas=0`
+```
+
+* Suspend all mon pods and osd pods as follows.
+
+```console
+MONS=<mon's deployments>
+OSDS=<osd's deployments>
+
+for mon in ${MONS}; do
+    kubectl -n ${NS} patch deployment ${mon} -p '{"spec": {"template": {"spec": {"containers": [{"name": "mon", "command": ["sleep", "infinity"], "args": []}]}}}}'
```

Comment:
This line gives an error:
```
Error from server (BadRequest): invalid character ']' after object key
```

Reply:
I guess that I made a mistake when copy-and paste the these steps. I'll check it again later.